### PR TITLE
Updated emoji mart package

### LINKED ISSUE #246: Update Emoji Picker package
@mohit-s96 _a Please update the emoji picker to the latest version and compare the changes in bundle size. IMO, the bundle size should reduce quite a bit. 
Also, there are other ways to reduce the bundle size of emoji-picker which are described in the package's Git repo. Please check and lmk if we need to discuss this in detail. 

cc: @labeebklatif @AbhayVAshokan

### PR DESCRIPTION
**Pull request by mohit-s96**
closes #246 

Updated the `emoji-mart` package to the latest version. The newest version had a lot of breaking changes. Also updated are the `webpack` related packages because the older versions were not working with the `emoji-mart` package. 

The update overall cuts down a little over 1MB from the build size.

## before the update

before_update

## after the update

after_update

### CODE REVIEW

**Comment by mohit-s96** [comment]
@AbhayVAshokan _a. Please review.

File: lib/components/Editor/CustomExtensions/Emoji/EmojiSuggestion/ExtensionConfig.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ const EmojiSuggestion = Node.create({
         ({ chain }) => {
           chain()
             .focus()
-            .insertContent(`${emoji?.native || ""} `)
+            .insertContent(`${emoji?.native || emoji?.skins[0]?.native || ""} `)
```
Comment (AbhayVAshokan) [inline lib/components/Editor/CustomExtensions/Emoji/EmojiSuggestion/ExtensionConfig.js:82]: I understand that `emoji.native` is no longer valid. Replace it with
```suggestion
            .insertContent(`${emoji?.skins[0]?.native || ""} `)
```

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@mohit-s96 _a please have a look at the changes. Also please apply these suggestions as well: https://vimeo.com/715270840/7e1b6f2fb1

**Comment by karthiknmenon** [comment]
@AbhayVAshokan _a Please take this up.

**Comment by AbhayVAshokan** [comment]
@labeebklatif _a please have a look.

**Comment by labeebklatif** [comment]
@AbhayVAshokan _a Please have a look at this [video](https://www.loom.com/share/e2a03c06c3e04de4ae6f05695d4fceb4)

**Pull request merged**